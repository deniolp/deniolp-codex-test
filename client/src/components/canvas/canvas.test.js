import React from 'react';
import renderer from 'react-test-renderer';

import Canvas from './canvas';

describe(`Canvas`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<Canvas value={`This is a value`} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
