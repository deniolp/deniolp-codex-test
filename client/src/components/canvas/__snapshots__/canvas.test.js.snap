// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canvas renders correctly 1`] = `
<textarea
  disabled="disabled"
  style={
    Object {
      "backgroundColor": "inherit",
      "border": 0,
      "color": "#000d33",
      "height": "65vh",
      "width": "100%",
    }
  }
  value="This is a value"
/>
`;
